client/partials/abtAppropriations.html

```diff
@@ -0,0 +1,3 @@
+<div class="abtAppropriations">
+    <h1>About the Appropriations Chair</h1>
+</div>
```

Comment:
Also, git doesn't like it when we don't have an empty newline at the end of a file. We might want to fix that to make git happy.